data/init.sql

```diff
@@ -25,6 +25,18 @@ CREATE TABLE IF NOT EXISTS networkconfigs(
       ip_sync_enabled BOOLEAN NOT NULL,
       CONSTRAINT networkprofilename UNIQUE(network_profile_name)
     );
+CREATE TABLE IF NOT EXISTS wirelessconfigs(
+      wireless_profile_name citext NOT NULL,
+      authentication_method integer,
+      encryption_method integer,
+      ssid varchar(32),
+      priority integer,
+      pskPassPhrase varchar(63),
+      linkPolicy int[],
```

Comment:
why are some of these underscored for spaces, and some camlCase? We should take a long hard look across all our database column names and decided on a standard and stick with it.